cellengine/utils/helpers.py

```diff
@@ -120,3 +131,33 @@ def datetime_to_timestamp(value: datetime) -> str:
     ``datetime`` objects.
     """
     return datetime.strftime(value, "%Y-%m-%dT%H:%M:%S.%fZ")
+
+
+def normalize(
+    value: float,
+    observed_min: float,
+    observed_max: float,
+    min: float,
+    max: float,
+):
+    return (max - min) / (observed_max - observed_min) * (value - observed_max) + max
+
+
+def flatten(d):
```

Comment:
I might be missing it, but I don't see where this is used. 🧐

Reply:
appears not to be :smile: